test/service_tests/user_config_test.dart

```diff
@@ -170,5 +245,31 @@ void main() async {
       expect(mockUser.authToken, newAuthToken);
       expect(mockUser.refreshToken, newRefreshToken);
     });
+
+    test('Test for saveCurrentOrgInHive method.', () async {
```

Comment:
How is this testing saveCurrentOrgInHive method ? Please explain.

Reply:
The test calls the `saveCurrentOrgInHive` method, storing the `_currentOrg` object in the `currentOrg` Hive box. The test assumes that if there's any problem while saving orgInfo into the Hive box, the `box.put` method will throw an exception, causing the test to fail. A successful test execution confirms the successful storage of orgInfo within the currentOrg Hive box.

Reply:
Alright. Add a comment explaining this.